src/extension/tags.c

```diff
@@ -799,15 +898,40 @@ static PHP_FUNCTION(datadog_appsec_track_user_login_success_event)
     }
     zend_array *meta_ht = Z_ARRVAL_P(meta);
 
-    // usr.id = <user_id>
-    _add_new_zstr_to_meta(meta_ht, _dd_tag_user_id, user_id, true);
+    if (user_id && ZSTR_LEN(user_id) > 0) {
+        dd_find_and_apply_verdict_for_user(user_id);
+        // usr.id = <user_id>
+        _add_new_zstr_to_meta(
+            meta_ht, _dd_tag_user_id, user_id, true, !automated);
```

Comment:
It might be more readable to have an `override` variable with a comment explaining why `!automated` makes sense instead of just using `!automated` everywhere.

Reply:
Ok